packages/headless/src/controllers/commerce/core/breadcrumb-manager/headless-core-breadcrumb-manager.ts

```diff
@@ -142,6 +146,7 @@ export function buildCoreBreadcrumbManager(
 
   const createBreadcrumb = (facet: AnyFacetResponse) => ({
     facetId: facet.facetId,
+    facetDisplayName: facet.displayName,
```

Comment:
Also out of scope, but this was a quick win. We need the facet display name to properly display breadcrumbs in a UI, otherwise they're just going to indicate the selected values with no further context, which can get confusing.